setup.py

```diff
@@ -0,0 +1,13 @@
+# pylint: disable=missing-module-docstring
+from setuptools import setup
+
+setup(
```

Comment:
Did you mean to fill the empty fields later?

Reply:
Yes, depending on whether we are interested in a package or not. For now I just needed a package to collect functions in.